env/Lib/site-packages/PIL/ImageMath.py

```
@@ -0,0 +1,78 @@
+
+    if options is None:  # Initialize options safely
+        options = {}
+
+    if options:
+        deprecate(
+            "ImageMath.lambda_eval options",
+            12,
+            "ImageMath.lambda_eval keyword arguments",
+        )
+
+    args: dict[str, Any] = ops.copy()
```

Ruff (0.8.2)

Undefined name `ops`

(F821)